telemetry/specs/INT.mdk

```diff
@@ -1596,6 +1844,18 @@ INT Metadata Header and Metadata Stack:
    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 `
 
```

Comment:
IMO it would be cleaner to add a section header. Change the title of the previous example, replacing "INT" with "INT-MD", then add a title here with "INT-MX", referring to the previous section for the VXLAN-GPE header. Start with the INT shim header with type = 3 and length = 3 here.

Reply:
Done